bmButtonHandlers.lua

```diff
@@ -164,10 +164,11 @@ function BMButtonHandlers.RenderButtonIcon(Button, cursorScreenPos, size)
     end
 
     local renderIconAnim = animItems
-    if iconType == nil or iconType == "Spell" then
+
+    if iconType == "Spell" then

```

Comment:
This is likely broken as nil is a valid case. which will now be either nil or '' with your changes...